packages/integration-sdk-runtime/src/execution/__tests__/jobState.test.ts

```diff
@@ -163,21 +163,21 @@ describe('#hasKey', () => {
     vol.reset();
   });
 
-  test('answers false when no entity or relationship added', async () => {
+  test('answers false when no entity or relationship added', () => {
     const jobState = createTestStepJobState();
-    expect(await jobState.hasKey('a')).toBeFalse();
+    expect(jobState.hasKey('a')).toBeFalse();
```

Comment:
This was from another PR, yes?

Reply:
Yes, this was changed by Austin and eslint started to complaint about it so I'm just adding this as additional